<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringPage</class>
 <widget class="QWidget" name="MonitoringPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="monitoringTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="computerTab">
    <attribute name="title">
     <string>Computer</string>
    </attribute>
    <widget class="QFrame" name="cpuFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>775</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="cpuBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>cpu info type</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="cpuPlot" native="true">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="ramFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>375</width>
       <height>301</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="ramLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>RAM</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="ramBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="ramDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>RAM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCustomPlot" name="ramPlot" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="diskFrame">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>220</y>
       <width>391</width>
       <height>301</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="diskLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>DISK</string>
      </property>
     </widget>
     <widget class="QComboBox" name="diskBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="diskBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QCustomPlot" name="diskPlot" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>351</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="softwareTab">
    <attribute name="title">
     <string>Software</string>
    </attribute>
    <widget class="QFrame" name="ramFrame_2">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>210</y>
       <width>381</width>
       <height>311</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="ramLabel_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>RAM</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="ramSelfBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>65</red>
            <green>255</green>
            <blue>112</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>65</red>
            <green>255</green>
            <blue>112</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>145</red>
            <green>141</green>
            <blue>126</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QCustomPlot" name="ramSelfPlot" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ramSelfDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(65, 255, 112);</string>
      </property>
      <property name="text">
       <string>RAM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="cpuFrame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>375</width>
       <height>311</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="cpuSelfBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>65</red>
            <green>255</green>
            <blue>112</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>65</red>
            <green>255</green>
            <blue>112</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>145</red>
            <green>141</green>
            <blue>126</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpuLabel_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>CPU</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="cpuSelfPlot" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="infoFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>765</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="cpuLabel_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Last Command</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Framework</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Executable Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="softwareVersion">
         <property name="text">
          <string>0.1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="frameworkVersion">
         <property name="text">
          <string>Qt</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="exeSize">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>406</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridData">
       <item row="0" column="0">
        <widget class="QLabel" name="frameworkVersion_2">
         <property name="text">
          <string>PolyCommand Send</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="basicSend">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="polySend">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="frameworkVersion_4">
         <property name="text">
          <string>BasicCommand Send</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="frameworkVersion_6">
         <property name="text">
          <string>BasicCommand Received</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="frameworkVersion_5">
         <property name="text">
          <string>PolyCommand Received</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="polyReceived">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="basicReceived">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="firmwareTab">
    <attribute name="title">
     <string>Firmware</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../externals/QCustomPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
